Description: replace some obsolete function in script
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-03-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -183,7 +183,7 @@
            sed 's/\([[0-9]]*\).\([[0-9]]*\).\([[0-9]]*\)/\3/'`
 
 AC_LANG_SAVE
-AC_LANG_C
+AC_LANG([C])
 AC_TRY_COMPILE([
 #include <alsa/asoundlib.h>
 ], [
